service/controller/clusterapi/v29/resource/tcnp/template/template_main_vpc_cidr.go

```diff
@@ -0,0 +1,11 @@
+package template
+
+const TemplateMainVPCCIDR = `
+{{- define "vpc_cidr" -}}
```

Comment:
This is missing in `render.go` to make the templating work.

Reply:
Right, I had to fix a couple of things.